teams/codebender.md

```diff
@@ -9,15 +9,51 @@ permalink: /teams/codebender/
  - Thng Kai Yuan ([@thngkaiyuan](http://cs3281-2.blogspot.sg/))
    - Technology Expertise: Information Security
  - Moon Seokmin ([@bsmmoon](http://moonmoon3281.blogspot.sg/))
-   - Technology Expertise:
+   - Technology Expertise: Web Crawling
```

Comment:
yes, the question is, how can it be phrased as a skill, like something that other people do not have.